_aggregations/bucket/children.md

```diff
@@ -2,134 +2,127 @@
 layout: default
 title: Children
 parent: Bucket aggregations
-grand_parent: Aggregations
 nav_order: 15
+redirect_from:
+  - /query-dsl/aggregations/bucket/children/
 ---
 
 # Children
 
-The `children` aggregation connects parent documents with their related child documents. This allows you to analyze relationships between different types of data in a single query, rather than needing to run multiple queries and combine the results manually.
+The `children` aggregation is a bucket aggregation that creates child buckets based on parent-child relationships defined in your documents. 
 
----
+The `children` aggregation works with the [join field type]({{site.url}}{{site.baseurl}}/field-types/supported-field-types/join/) to aggregate child documents that are associated with parent documents.
+
+The `children` aggregation provides inverse functionality to the [parent aggregation]({{site.url}}{{site.baseurl}}/aggregations/bucket/parent/). The parent aggregation identifies parent documents based on matching child documents; the children aggregation identifies child documents based on matching parent documents.
+
+
+## Parameters
 
-## Example index, sample data, and children aggregation query
+The `children` aggregation takes the following parameters:
 
-For example, if you have a parent-child relationship between authors, posts, and comments, you can analyze the relationships between the different data types (`authors`, `posts`, and `comments`) in a single query. 
+| Parameter             | Required/Optional | Data type       | Description |
+| :--                   | :--               | :--             | :--         |
+| `type`                | Required          | String          | The name of the child type from the join field. This identifies the parent-child relationship to use. |
+| `aggs`                | Optional          | Object          | Sub-aggregations to run on the child documents. |
```

Comment:
```suggestion
```

I think this aggs is general for any type of bucket aggregation, so better to not showing here but in a general doc page for bucket aggregation